workflow/sample_files/base-hvac-geothermal-loop-borefield-configuration.xml

```diff
@@ -0,0 +1,565 @@
+<?xml version='1.0' encoding='UTF-8'?>
```

Comment:
We probably don't need _so many_ extra geothermal-loop sample files. Can we somehow condense them down?

Reply:
Narrowed down to just 2 new sample files:
* `base-hvac-geothermal-loop.xml`
* `base-misc-ground-diffusivity.xml`